pkgs/development/libraries/libcs50/default.nix

```diff
@@ -11,18 +11,22 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-A4CEU5wfwykVTDIsKZnQ8co+6RwBGYGZEZxRFzQTKBI=";
   };
 
+  nativeBuildInputs = lib.optionals stdenv.isDarwin [ fixDarwinDylibNames ];
+
   installPhase = ''
     runHook preInstall
-    mkdir $out
-    cp -R build/lib $out/lib
-    cp -R build/include $out/include
+    make install DESTDIR="$out"
+  '' + lib.optionalString stdenv.isLinux ''
     ln -sf $out/lib/libcs50.so.11.0.2 $out/lib/libcs50.so.11
+  '' + ''
     runHook postInstall
   '';
```

Comment:
```suggestion
  installFlags = [ "DESTDIR=$(out)" ];

  postInstall = lib.optionalString stdenv.isLinux ''
    ln -sf $out/lib/libcs50.so.${finalAttrs.version} $out/lib/libcs50.so.${lib.versions.major finalAttrs.version}
  '';
```

Reply:
sorry I'm just playing around with some stuff, using this as a sandbox. Let me revert to draft.